tests/VM_with_planetary_network/main.tf

```diff
@@ -0,0 +1,53 @@
+variable "public_key" {
+  type = string
+}
+
+terraform {
+  required_providers {
+    grid = {
+      source = "threefoldtech/grid"
+      version = "0.1.2"
```

Comment:
remove the version from the tests to always use latest published version
do this for all test cases